lib/bootboot/bundler_patch.rb

```diff
@@ -54,3 +54,19 @@ def enable_dual_booting
     Bundler::SharedHelpers.singleton_class.prepend(SharedHelpersPatch)
   end
 end
+
+module AppCachePatch
+  def app_cache(custom_path = nil)
+    path = custom_path || root
+    Pathname.new(path).join("#{settings.app_cache_path}-next")
+  end
+end
+
+module DslPatch
+  def enable_dual_booting
```

Comment:
Do we have to override this method? Can't we just add this line `Bundler.singleton_class.prepend(AppCachePatch)` 
 inside the method directly

Reply:
I have updated it to patch `Bundler::Settings` directly

```ruby
Bundler::Settings.prepend(Module.new do
   def app_cache_path
     'vendor/cache-next'
   end
end)
```